site/docs/components/autocomplete.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: Autocomplete
+navTitle: Autocomplete (not built)
+summaryParagraph: Autocomplete automatically completes typed user input with matching results from a larger data set. Autosuggest breaks beyond the input provided to suggest alternative, relevant answers.
+tags: ["Autosuggest", "Live filter", "Filters", "Search", "Top bar", "Text input"]
+needToKnow:
+- 'We separate Search Field into a separate component, because it may be used for live-filtering without showing autocompleted results.'
+- 'Unlike a Select (single- or multi-) that lets you select options from an options menu, the Autocomplete component has other behavior beyond choosing an option as the value for a setting, such as linking to other content.'
```

Comment:
Does this refer to a _select_ component or the _dropdown_ component used with a select?

Reply:
It refers to the entire select component. What could make this more useful?

Reply:
Now that I'm re-reading it I'm not sure what I was getting at here 🤦‍♂️Let's drop it :)